src/devices/Dhtxx/samples/DhtSensor.sample.cs

```diff
@@ -16,27 +13,19 @@ static void Main(string[] args)
     {
         Console.WriteLine("Hello DHT!");
 
-        DHTSensor dht = new DHTSensor(26, DhtType.Dht22);
+        // Init DHT12 through I2C
```

Comment:
would be nice to put this in a separate function and just comment out the call so that we at least know this builds